src/worker/token.worker.ts

```diff
@@ -20,8 +20,27 @@ const deleteRefreshToken = (audience: string, scope: string) =>
 const wait = (time: number) =>
   new Promise(resolve => setTimeout(resolve, time));
 
+const parseFormData = (formData: string) => {
+  const queryParams = formData.split('&');
+  const parsedQuery: any = {};
+
+  queryParams.forEach(param => {
+    const [key, val] = param.split('=');
+    parsedQuery[key] = decodeURIComponent(val);
+  });
+
+  return parsedQuery;
+};
+
+const createFormData = (params: any) => {
+  return Object.keys(params)
+    .filter(k => typeof params[k] !== 'undefined')
+    .map(k => encodeURIComponent(k) + '=' + encodeURIComponent(params[k]))
+    .join('&');
+};
+
```

Comment:
This could have been included from `utils` but doing so adds another 2.5kb to the bundle, as the module is copied.